tests/test_simpleTable.py

```diff
@@ -799,6 +799,26 @@ def testDelete(self):
         with self.assertRaises(IndexError):
             del catalog[50]
 
+    def testSetFlagColumn(self):
+        schema = lsst.afw.table.Schema()
+        key = schema.addField("a", type="Flag", doc="doc for 'a'")
```

Comment:
I wonder if we should add another layer to this test, with a `b` flag as well as an `a` flag, to ensure that the bit-packing behind the scenes is doing what we think it should be doing.  (However, I did confirm that this does work with multiple flags).

Reply:
The bit-packing is implemented in a level below this that has its own tests, so I think one is sufficient.